compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -174,6 +174,21 @@ struct InsertQuantizeOp final : public luci::CircleNodeMutableVisitor<void>
     insert_out_quantize(node);                                           \
   }
 
+  // INPUT_NAME is the only activation of NODE
+#define INSERT_QUANTIZE_TO_UNARY_MULTI_OUTPUT_OP(NODE, INPUT_NAME)            \
+  void visit(NODE *node)                                                      \
+  {                                                                           \
+    if (auto input_quant = create_in_quantize(node->INPUT_NAME(), node))      \
+      node->INPUT_NAME(input_quant);                                          \
+                                                                              \
+    auto spilt_out_nodes = loco::succs(node);                                 \
```

Comment:
This macro is not specific to split. 
```suggestion
    auto succs = loco::succs(node);                                 \
```